doc/conf.py

```diff
@@ -47,48 +51,56 @@
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
-# We use the version module generated by setup.py so that we are always
-# consistent with the version specified in that file.
+# We use the version module in src/psyclone. However, rather than importing
+# it (which would require that PSyclone be installed first), we read it
+# using execfile().
 # 'version' is the short X.Y version and 'release' is the full version,
 # including any alpha/beta/rc tags.
-from psyclone.version import version as release, short_version as version
+# We are in the doc directory but need to read version.py from
+# src/psyclone
+BASE_PATH = os.path.dirname(os.path.abspath(__file__))
+BASE_PATH = os.path.dirname(BASE_PATH)
+execfile(os.path.join(BASE_PATH, "src", "psyclone", "version.py"))
```

Comment:
We now read version information from a file. Much better. The only issue (as already privately discussed) is that execfile is not supported in Python3, but we can cross that bridge when we come to it.